adafruit_bme680.py

```diff
@@ -537,86 +502,75 @@ def _set_heatr_conf(self, heater_temp: UINT16, heater_time: UINT16) -> None:
             ctrl_gas_data_1 = bme_set_bits(
                 ctrl_gas_data_1, _BME68X_RUN_GAS_MSK, _BME68X_RUN_GAS_POS, run_gas
             )
-            self._write(ctrl_gas_addr_0, [ctrl_gas_data_0])
-            self._write(ctrl_gas_addr_1, [ctrl_gas_data_1])
+            self._write(_BME68X_REG_CTRL_GAS_0, [ctrl_gas_data_0])
+            self._write(_BME68X_REG_CTRL_GAS_1, [ctrl_gas_data_1])
             # HELP check this
             self._set_op_mode(_BME68X_FORCED_MODE)
-        except GasHeaterException as exc:
+        except OSError as exc:
             self._set_op_mode(_BME68X_FORCED_MODE)
             raise exc
```

Comment:
I think this might be simplified to:

```suggestion
        finally:
            self._set_op(_BME68X_FORCED_MODE)
```

Reply:
I tried this but pylint reports this error:
```
adafruit_bme680.py:509:12: E1101: Instance of 'Adafruit_BME680' has no '_set_op' member (no-member)
```

I'm guess that it's meant to be `_set_op_mode()` I'll commit it with that change.